src/api/firebase.js

```diff
@@ -84,3 +90,17 @@ export async function deleteItem() {
 export async function createNewList(newListToken) {
 	await addDoc(collection(db, newListToken), {});
 }
+
+/** This function queries the database for an existing shopping list collection */
+export async function queryForCollection(listId) {
+	// Create a reference to the shopping list collection
+	const listCollectionRef = collection(db, listId);
+
+	// Create a query against the shopping list collection
+	const q = query(listCollectionRef);
```

Comment:
@djtaylor8 I'm not sure that it should! Adding `await` here causes VSCode to say it doesn't have an effect on the operation. Maybe because `query` in this case is not performed asynchronously? But the line right below it (`getDocs`) does indeed need the `await`.